validator/src/main/java/com/google/daq/mqtt/sequencer/SequenceBase.java

```diff
@@ -384,6 +386,7 @@ private static void setupSequencer() {
     String registrySuffix = exeConfig.registry_suffix;
     altRegistry = SiteModel.getRegistryActual(udmiNamespace, altRegistryId, registrySuffix);
     altClient = getAlternateClient();
+    ifNotNullThen(altClient, IotReflectorClient::activate);
```

Comment:
I also want to understand why we are now calling for client activation and why it worked before without it